eslint-plugin-expensify/tests/prefer-at.test.js

```diff
@@ -0,0 +1,37 @@
+const RuleTester = require('eslint').RuleTester;
+const rule = require('../prefer-at');
+const message = require('../CONST').MESSAGE.PREFER_AT;
+
+const ruleTester = new RuleTester({
+    parserOptions: {
+        ecmaVersion: 6,
+        sourceType: 'module',
+    },
+});
+
+ruleTester.run('prefer-at', rule, {
+    valid: [
+        {
+            code: 'example.at(0)',
+        },
+        {
+            code: 'test.at(1)',
+        },
+    ],
+    invalid: [
+        {
+            code: 'example[0]',
```

Comment:
I think we can add more tests here, like
```
array[array.length - 1];
array[0][array[0].length - 1];
array[(array.length - 1) / 2];
```

Let me try it in codebase and feedback later.